code/modules/mob/living/carbon/monkey/inventory.dm

```diff
@@ -7,12 +7,8 @@
 				if (!( target.wear_mask ))
 					qdel(src)
 					return
-			if("l_hand")
-				if (!( target.l_hand ))
-					qdel(src)
-					return
-			if("r_hand")
-				if (!( target.r_hand ))
+			if("hand")
```

Comment:
pls these are everywhere in the code, and there are already 115 files changed + conflicts